db/migrations/postgres/000054_create_blockchainevents_table.up.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+CREATE TABLE blockchainevents (
```

Comment:
Think we need an index on `tx_id` for this one

Reply:
and `id`

Reply:
I see that the `onchain-logic` branch has these indexes now (they are on `contractevents` but I just renamed that table):
```
CREATE INDEX blockchainevents_name ON blockchainevents(namespace,name);
CREATE INDEX blockchainevents_timestamp ON blockchainevents(timestamp);
CREATE INDEX blockchainevents_subscription_id ON blockchainevents(subscription_id);
```

If we add 2 more indexes, is that a bit much for a potentially high-write-volume table?

Reply:
I do think the ones you mentioned are useful... I guess just wondering if I should also prune any others? Possibly `name`?

Reply:
Think we go with this for now, and test - can't see any obvious candidates to cull.

Reply:
In the end I did remove `namespace,name`. It's easy enough to add back later if we want.